packages/hash/frontend/src/theme/components/inputs/MuiButtonThemeOptions.ts

```diff
@@ -14,114 +17,231 @@ export const MuiButtonThemeOptions: Components<Theme>["MuiButton"] = {
     root: ({ ownerState, theme }) => {
       const { variant, size } = ownerState;
 
-      // The base CSS styling applied to the button
-      const baseStyles: CSSObject = {
+      /** ====================== INITIAL Button styles ============================= */
+
+      // base CSS styling applied to the button
+      let baseStyles: CSSObject = {
         textTransform: "none",
-        lineHeight: "24px",
+
+        [`& > .${buttonClasses.startIcon}, & > .${buttonClasses.endIcon}`]: {
+          color: "currentColor",
+        },
       };
 
-      // The :before CSS styling applied to the button
-      const beforeStyles: CSSObject = {
+      // :before CSS styling applied to the button
+      let beforeStyles: CSSObject = {
         content: `""`,
         borderRadius: "inherit",
         position: "absolute",
         width: "100%",
         height: "100%",
-        border: "1px solid transparent",
       };
 
-      // The :hover CSS styling applied to the button
-      const hoverStyles: CSSObject = {};
+      // :hover CSS styling applied to the button
+      let hoverStyles: CSSObject = {};
 
-      // The :hover:before CSS styling applied to the button
-      const hoverBeforeStyles: CSSObject = {};
+      // :hover:before CSS styling applied to the button
+      let hoverBeforeStyles: CSSObject = {};
 
-      // The .Mui-disabled CSS styling applied to the button
-      const disabledStyles: CSSObject = {};
+      // .Mui-disabled CSS styling applied to the button
+      const disabledStyles: CSSObject = {
+        background: theme.palette.gray[20],
+        color: theme.palette.gray[50],
+      };
 
       // The :active CSS styling applied to the button
       const activeStyles: CSSObject = {};
 
       // The :focus CSS styling applied to the button
-      const focusStyles: CSSObject = { outline: "none" };
+      let focusStyles: CSSObject = { outline: "none" };
 
-      // The :focus-visible:after CSS styling applied to the button
-      const focusVisibleAfterStyles: CSSObject = {
+      // The :focus:after CSS styling applied to the button
+      const focusAfterStyles: CSSObject = {
         content: `""`,
         position: "absolute",
         left: -buttonFocusBorderOffset,
         top: -buttonFocusBorderOffset,
         bottom: -buttonFocusBorderOffset,
         right: -buttonFocusBorderOffset,
         border: `${buttonFocusBorderWidth}px solid`,
```

Comment:
**Blocking nit pick:** I think the designs have a slightly less wide border-width for non-large buttons

Reply:
Maybe capture that in a follow-up task then (alongside verifying whether the focus state should be `:focus` or `:focus-visible`)

Reply:
Noted. Created a task to track that => [internal link](https://app.asana.com/0/1201959586244671/1201970018157947)